config/clusters/dandi/common.values.yaml

```diff
@@ -0,0 +1,145 @@
+jupyterhub:
+  hub:
+    config:
+      JupyterHub:
+        authenticator_class: github
+      Authenticator:
+        enable_auth_state: true
+        admin_users:
+          - kabilar
+          - aaronkanzer
+          - asmacdo
+          - satra
+          - yarikoptic
+          - waxlamp
+  custom:
+    2i2c:
+      add_staff_user_ids_to_admin_users: true
+      add_staff_user_ids_of_type: "github"
+    jupyterhubConfigurator:
+      enabled: false
+    homepage:
+      templateVars:
+        org:
+          logo_url: "https://raw.githubusercontent.com/dandi/artwork/3f287d3ae53154a66f8b50711549740719a23fdb/pics/dandi-logo.svg"
+          url: "https://dandiarchive.org/"
+        designed_by:
+          name: 2i2c
+          url: https://2i2c.org
+        operated_by:
+          name: 2i2c
+          url: https://2i2c.org
+        funded_by:
+          name: "DANDI (MIT Brain)"
+          url: "https://dandiarchive.org/"
+  scheduling:
+    userScheduler:
+      enabled: true
+  singleuser:
+    profileList:
+      - display_name: "DANDI (CPU)"
+        description: "Default DANDI image with JupyterLab"
+        default: true
+        kubespawner_override:
+          image: dandiarchive/dandihub:latest
+          image_pull_policy: Always
+          default_url: /lab
+        profile_options: &profile_options_cpu
+          requests:
+            display_name: Resource Allocation
+            choices:
+              mem_3_7:
+                display_name: 3.7 GB RAM, upto 3.7 CPUs
+                kubespawner_override:
+                  mem_guarantee: 3982682624
+                  mem_limit: 3982682624
+                  cpu_guarantee: 0.46875
+                  cpu_limit: 3.75
+                  node_selector:
+                    node.kubernetes.io/instance-type: r5.xlarge
+                default: true
+              mem_7_4:
+                display_name: 7.4 GB RAM, upto 3.7 CPUs
+                kubespawner_override:
+                  mem_guarantee: 7965365248
+                  mem_limit: 7965365248
+                  cpu_guarantee: 0.9375
+                  cpu_limit: 3.75
+                  node_selector:
+                    node.kubernetes.io/instance-type: r5.xlarge
+              mem_14_8:
+                display_name: 14.8 GB RAM, upto 3.7 CPUs
+                kubespawner_override:
+                  mem_guarantee: 15930730496
+                  mem_limit: 15930730496
+                  cpu_guarantee: 1.875
+                  cpu_limit: 3.75
+                  node_selector:
+                    node.kubernetes.io/instance-type: r5.xlarge
+              mem_29_7:
+                display_name: 29.7 GB RAM, upto 3.7 CPUs
+                kubespawner_override:
+                  mem_guarantee: 31861460992
+                  mem_limit: 31861460992
+                  cpu_guarantee: 3.75
+                  cpu_limit: 3.75
+                  node_selector:
+                    node.kubernetes.io/instance-type: r5.xlarge
```

Comment:
I went and re-read the requirements document again, and noticed that they wanted options upto 512GB of RAM. So I've updated the issue with a few more options here. Should be copy-pasteable.